README.md

```diff
@@ -11,6 +11,20 @@ This collection has been tested against FRR 6.0.
 The FRR collection supports ``network_cli`` connections.
 
 ## Included content
+<!--start collection content-->
+## Cliconf plugins
+Name | Description
+--- | ---
+[frr.frr.frr](https://github.com/ansible-collections/frr.frr/blob/master/docs/frr.frr.frr.rst)|Use frr cliconf to run command on Free Range Routing platform
+## Terminal plugins
+Name | Description
+--- | ---
```

Comment:
Hmmm, I wonder whether we should drop `terminal` from add_docs or add DOCUMENTATION strings to them

Reply:
Good catch, I don't see why terminal plugins couldn't have a doc string.